### ISSUE TITLE
Cursor color

### ISSUE THREAD

**Issue by arcas0803** [Alexqwesa/app_bar_with_search_switch#6]
Hi!

Amazing work of this package.

Can you add the option to pass cursor color throw arguments instead of taking it from theme. Envolving the widget in TextSelectionTheme is not working so ...

Thanks

**Comment by Alexqwesa** [comment]
it is related to this bug, https://github.com/flutter/flutter/issues/74890 - here is  also workaround, but i am not sure it works in all cases

**Comment by Alexqwesa** [comment]
it had been fixed in upstream

**Comment by dickermoshe** [comment]
The default of `keepAppBarColors: true` results in a light themed cursor color and it cant be changed by 
```
    return TextSelectionTheme(
      data: TextSelectionThemeData(cursorColor: Colors.black),
```
I set it to false.
Dont know if this is a bug, will leave here if anyone has this issue.

**Comment by Alexqwesa** [comment]
thanks for mentioning this, i will fix it soon...

**Comment by Alexqwesa** [comment]
keepAppBarColors: true  -  was useful  for material theme 2, but not now, should be fixed in v1.8.0(2.0.0-dev.18)

https://pub.dev/packages/app_bar_with_search_switch/changelog
